buku.py

```diff
@@ -3219,6 +3203,45 @@ def prompt(obj, results, noninteractive=False, deep=False, subprompt=False, sugg
                 break
 
 
+def print_rec_with_filter(records, field_filter):
```

Comment:
make the default field filter 0 and call `print_single_rec` within it when filter is 0.